controllers/space/space_controller.go

```diff
@@ -340,7 +340,10 @@ func (r *Reconciler) manageNSTemplateSet(logger logr.Logger, space *toolchainv1a
 			}
 			nsTmplSet = NewNSTemplateSet(memberCluster.OperatorNamespace, space, spaceBindings.Items, tmplTier)
 			if err := memberCluster.Client.Create(context.TODO(), nsTmplSet); err != nil {
-				logger.Error(err, "failed to create NSTemplateSet on target member cluster")
+				if errors.IsAlreadyExists(err) {
+					// requeue immediately, there's probably a race condition between the host client cache and the member cluster state
+					return nsTmplSet, norequeue, nil
```

Comment:
```suggestion
					return nsTmplSet, requeueDelay, nil
```
should it return requeueDelay instead of norequeue ?

Reply:
why? I'm not sure we need to wait any longer. Unless we need to give time to the client to update its cache?

Reply:
Sorry, my suggestion was incorrect as well!

Don't you need to return an error at least, so that the reconcile is triggered ?

Reply:
you're right, I did not realize that `norequeue` was the wrong value to return here. I read that its value was zero but I missed the check upstream 🤦‍♂️ 
I applied your suggestion in 1af8c57

Reply:
thanks for your patience @mfrancisc !

Reply:
No worries, it was unclear also for me what happened with the upstream functions !